cdc-accelerator/feature/acl.js

```diff
@@ -29,21 +28,29 @@ export default class ACL extends Feature {
             }
         }
     }
-
     build(directory) {
-        const buildFeaturePath = path.join(directory, this.getName())
-        clearDirectoryContents(buildFeaturePath)
-        const srcFeaturePath = buildFeaturePath.replace(BUILD_DIRECTORY, SRC_DIRECTORY)
+        const srcFeaturePath = directory.replace(BUILD_DIRECTORY, SRC_DIRECTORY)
+        const srcAclPath = path.join(srcFeaturePath, this.getName())
+        const buildPath = path.join(directory, this.getName())
+        this.createDirectoryIfNotExists(buildPath)
+        fs.readdirSync(srcAclPath).forEach((aclFile) => {
+            const aclData = fs.readFileSync(path.join(srcAclPath, aclFile), { encoding: 'utf8' })
+            fs.writeFileSync(path.join(buildPath, aclFile), aclData)
+        })
+    }
+    reset() {
+        //This is supposed to be empty
```

Comment:
Change the comment to something like 
// The method is empty because there is nothing left to do after PermissionGroups.reset is executed